psalm-baseline.xml

```diff
@@ -3833,8 +3758,16 @@
     <UndefinedInterfaceMethod occurrences="1">
       <code>plugin</code>
     </UndefinedInterfaceMethod>
-    <UnevaluatedCode occurrences="1">
+    <UnevaluatedCode occurrences="11">
```

Comment:
Some of these `<UnevaluatedCode` issues appearing, but they don't seem to be problematic: baseline got reduced in size anyway